CONTRIBUTING.md

```diff
@@ -81,6 +83,13 @@ https://github.com/pytorch/pytorch#from-source
 
 To develop PyTorch on your machine, here are some tips:
 
+### Prerequisites
+* CMake. We recommend installing it with [Homebrew](https://brew.sh/)
+with `brew install cmake` if you are developing on MacOS or Linux system.
```

Comment:
This seems wrong. CMake should better be installed from [PyPI](https://pypi.org/project/cmake/)